README.md

```diff
@@ -26,21 +25,41 @@ The [DevSecOps Example](https://github.com/GSA/devsecops-example) is a good star
 
 #### Terraform Modules
 
-* [Base Infrastructure](https://github.com/GSA/DevSecOps-Infrastructure)
+* [Base Infrastructure](https://github.com/GSA/DevSecOps)
 * [EKK (logging) stack](https://github.com/GSA/devsecops-ekk-stack)
 * [Jenkins](https://github.com/GSA/jenkins-deploy)
 * [AWS VPC flow logs](https://github.com/GSA/terraform-vpc-flow-log)
 
 #### Ansible Roles
 
 * [Jenkins](https://github.com/GSA/jenkins-deploy)
+* [Nessus](https://github.helix.gsa.gov/GSASecOps/ansible-nessus-agent)
 * [NGINX HTTPS proxy](https://github.com/GSA/ansible-https-proxy)
-
-#### Security Hardening
-
-These configure base operating systems in line with the Hardening Guides linked above, to the greatest extent possible.
-* [RHEL 6](https://github.com/GSA/ansible-os-rhel-6)
-* [RHEL 7](https://github.com/GSA/ansible-os-rhel-7)
-* [Ubuntu 14](https://github.com/GSA/ansible-os-ubuntu-14)
-* [Ubuntu 16](https://github.com/GSA/ansible-os-ubuntu-16)
-* [GPOs and INF for server 2008/2012](https://github.com/GSA/ISE-Security-Benchmark-GPOs)
+* [OSSEC][OSSEC]
+
+### By operating system
+
+Unless there's a good reason to go with something else, the following should be used on every server:
+
+Requirement | Linux | Windows
+--- | --- | ---
+Activity monitoring | [OSSEC][OSSEC] | ?
```

Comment:
What's the OSSEC equivalent for Windows?

Reply:
OSSEC has both windows and linux version.

Reply:
Ah, I see. Eliminating the source of my confusion in https://github.com/ossec/ossec.github.io/pull/5.